packages/@react-spectrum/progress/src/ProgressBarBase.tsx

```diff
@@ -77,7 +77,8 @@ function ProgressBarBase(props: ProgressBarBaseProps, ref: DOMRef<HTMLDivElement
           barClassName,
           styleProps.className
         )
-      }>
+      }
+      style={{minWidth: '-moz-fit-content'}}>
```

Comment:
probably should add a comment to explain this...

Reply:
Also I think this is overriding `styleProps` from above?

Reply:
mm it is, what do we want to do should someone pass in minWidth, do we want ours to win? probably not